lib/aerosol/launch_configuration.rb

```diff
@@ -90,6 +92,12 @@ def to_s
 }}
   end
 
+  def corrected_user_data
+    realized_user_data = user_data.is_a?(Proc) ? user_data.call : user_data
```

Comment:
Fair enough, forgot that since the proc will (almost definitely) be defined within the context of the DSL object, it will have access to the meta_data. :+1: